webpack/assets/javascripts/react_app/components/ModelsTable/ModelsTable.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { LoadingState } from 'patternfly-react';
+import PropTypes from 'prop-types';
+import { Table } from '../common/table';
+import { STATUS } from '../../constants';
+import MessageBox from '../common/MessageBox';
+import { translate as __ } from '../../common/I18n';
+import createModelsTableSchema from './ModelsTableSchema';
+import { getURIQuery } from '../../common/helpers';
+import Pagination from '../Pagination/Pagination';
+
+class ModelsTable extends React.Component {
+  componentDidMount() {
+    this.props.getTableItems('models', getURIQuery(window.location.href));
```

Comment:
we started to use more objects destruction in our components:
e.g: `const { getTableItems } = this.props`

if you want to get the query from the url, you could use the URI library which is already used in the project,
https://medialize.github.io/URI.js/docs.html#accessors-search